### PR TITLE
WW-15 | Better logging

### PR DESCRIPTION
**Pull request by rybmat**
Ticket: https://wikia-inc.atlassian.net/browse/WW-15

Since current logging is not very descriptive, I'd like to add some more information to logs in order to dig deeper into the bug.

@Wikia/west-wing

### CODE REVIEW

File: extensions/VisualEditor/ApiVisualEditor.php [hunk 1]
```diff
@@ -125,7 +125,9 @@ protected function requestParsoid( $method, $title, $params ) {
 			}
 			\Wikia\Logger\WikiaLogger::instance()->error( 'ApiVisualEditor_requestParsoid', [
 				'method' => $method,
-				'error' => 'parsoidserver-' . $code
+				'error' => 'parsoidserver-' . $code,
```
Comment (sebastianmarzjan) [inline extensions/VisualEditor/ApiVisualEditor.php]: Would it make sense to rename it to error_code and remove the 'parsoidserver-' part?

File: extensions/VisualEditor/ApiVisualEditor.php [hunk 2]
```diff
@@ -125,7 +125,9 @@ protected function requestParsoid( $method, $title, $params ) {
...
+				'errors' => json_encode($errors),
```
Comment (sebastianmarzjan) [inline extensions/VisualEditor/ApiVisualEditor.php]: Maybe error_messages ?

File: extensions/VisualEditor/ApiVisualEditor.php [hunk 3]
```diff
@@ -125,7 +125,9 @@ protected function requestParsoid( $method, $title, $params ) {
...
+			    'status_code' => $req->getStatus()
```
Comment (sebastianmarzjan) [inline extensions/VisualEditor/ApiVisualEditor.php]: Spacing :)

File: extensions/VisualEditor/ApiVisualEditor.php (hunk 1 above)
Comment (rybmat) [inline extensions/VisualEditor/ApiVisualEditor.php]: error_code may be missleading in my opinion as $code in this place has always value 'http-bad-status'

**Comment by sebastianmarzjan** [comment]
LGTM

File: extensions/VisualEditor/ApiVisualEditor.php (hunk 1 above)
Comment (rybmat) [inline extensions/VisualEditor/ApiVisualEditor.php]: actually we could get rid of this whole line...

Comment (sebastianmarzjan) [inline extensions/VisualEditor/ApiVisualEditor.php]: Go for it :)

**Pull request merged**